content/how-to/account-management/change-account-information/index.md

```diff
@@ -15,30 +15,41 @@ slug: change-account-information
 This article describes the various administrative account changes that you can
 request and the appropriate steps needed to complete them.
 
-### Dedicated customers
+### Dedicated Customers
 
-Dedicated customers can download the **Primary Contact Change** form in the
-[MyRackspace Control Panel](https://login.rackspace.com). For instructions
-about how to find the form, see the **Dedicated customers** section of [Change
-to a new primary contact](/support/how-to/change-to-new-primary-contact).
+You can download the **Primary Contact Change** form by using the following steps: 
 
-If you are the current primary contact for your account, you can update the
-primary contact information by selecting and editing the **Primary Contact
-Information** in your **Account Settings**.
+**Step 1.** Log in to the Rackspace Technology Portal <https://login.rackspace.com/>.
 
-Request the **Service Transfer** form from your account manager.
+**Step 2.** In the top navigation panel, select **Account >> Docs and Forms**.
+
+<img width="359" alt="Acc Docs   forms" src="Acc Docs & forms.png">
+
+**Step 3.** Under **Compliance Documents and Forms** click **Primary Contact Change Form**. Download the **Primary Contact Change Form** and save it. 
+
+<img width="324" alt="Change Primary contact form" src="Change Primary contact form.png">
+
+
+You can request the **Service Transfer** form from your account manager.
+
+
+### Cloud Customers
+
+Cloud customers can find links to the forms described in this article by using the following steps:
+
+**Step 1.** Log in to the Rackspace Technology Portal <https://login.rackspace.com/>. 
+
+**Step 2.** In the top navigation panel, select **Account >> Docs and Forms**.
+
+<img width="375" alt="Acc Docs   forms cloud" src="Acc Docs & forms cloud.png">
```

Comment:
image names should be lower case and no spaces

Reply:
spell out the text in the alt tag, and remove space in alt tag between "Docs   forms"